packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -1687,7 +1764,7 @@ class LayerLink {
 ///
 /// This layer collapses the accumulated offset into a transform and passes
 /// [Offset.zero] to its child layers in the [addToScene]/[addChildrenToScene]
-/// methods, so that [applyTransform] will work reliably.
+/// methods, so that [applyTransform] works reliably.
```

Comment:
generally speaking I think the flutter voice would use `will work` here (conceptually the method works when called, and will be called in the future, as opposed to working in general).

That said, feel free to leave this, no big deal.

Reply:
Reverted.